zen/pkm/scheduler.py

```
@@ -0,0 +1,401 @@
+        try:
+            async with GeminiExtractor(self.config) as extractor:
+                result = await extractor.extract_conversations(
+                    max_conversations=self.config.max_conversations_per_run
+                )
+                
+                if result.success:
+                    console.print(f"[green]✓[/green] Extracted {result.conversations_extracted} conversations")
+                else:
+                    console.print(f"[red]✗[/red] Extraction failed: {', '.join(result.errors)}")
+        
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)